bootstrap-orbit.ps1

```diff
@@ -35,19 +35,19 @@ You can call 'pip3 show -f conan' to figure out where conan.exe was placed.
 } else {
   Write-Host "Conan found. Checking version..."
 
-  $conan_version = (conan --version).split(" ")[2]
+  $conan_version = (& $conan.Path --version).split(" ")[2]
   $conan_version_major = $conan_version.split(".")[0] -as [int]
   $conan_version_minor = $conan_version.split(".")[1] -as [int]
 
   $conan_version_major_required = 1
-  $conan_version_minor_min = 27
+  $conan_version_minor_min = 29
 
   if ($conan_version_major -eq $conan_version_major_required -and $conan_version_minor -lt $conan_version_minor_min) {
     Write-Host "Your conan version $conan_version is too old. Let's try to update it."
 
     Try {
       $pip3 = Get-Command pip3
-      Start-Process -FilePath $pip3.Path -ArgumentList "install","--upgrade","conan" -Wait -NoNewWindow -PassThru
+      & $pip3.Path install --upgrade conan==1.29.2
     } Catch {
```

Comment:
I am no powershell expert, so forgive me if I am missing something. This here seems at least inconsistent with the calls to pip3 in line 22. 
Does pip3 install throw an error on fail that will be caught here, or do you have to check $LastExitCode as in line 22. Aka why don't you use TryCatch in line 22?

Reply:
There are two things happening here. The Try-Catch is NOT catching an install error. It's catching when the Get-Command does not find pip3 and `$pip3` will be null, so `$pip3.Path` throws.

The return code is ignored here on purpose for compatibility reasons.

In Line 22 it different because `$pip3` can't be null. This is checked before.

Reply:
> The return code is ignored here on purpose for compatibility reasons.
Sorry that I have to ask again, I don't understand this.. what compatibility?

If pip3 is available, but the install --upgrade fails, this will not be detected (I just checked it)

Reply:
Yes. Exactly. That is what I was saying. There was a bug about that. If I remember correctly this was coming from Pierric. pip3 was always failing due to some permission problem. But the conan installation was fine, somehow. And since the only point here is to provide a better error message to the user we decided to leave it like so.

Reply:
Ah okay. Now it makes sense. Thank you for taking the time to explain